pkg/pillar/scripts/device-steps.sh

```diff
@@ -21,8 +21,7 @@ TPM_DEVICE_PATH="/dev/tpmrm0"
 PATH=$BINDIR:$PATH
 
 echo "$(date -Ins -u) Starting device-steps.sh"
-echo "$(date -Ins -u) EVE version: $(cat /etc/eve-release)"
-cp -p /etc/eve-release /run/eve-release
+echo "$(date -Ins -u) EVE version: $(cat /run/eve-release)"
```

Comment:
cp -p /etc/eve-release /run/eve-release

-- IS this no longer needed? Just want to make sure this is not accidental..

Reply:
This is moved to another container (dom0-ztools)

Reply:
Cool.. Thanks 👍